project-1_test/test_main.py

```diff
@@ -0,0 +1,81 @@
+from main import *
```

Comment:
Kindly ensure you follow the recommendation:

*At least 3 test cases and 2 assertions per test case*

All your test cases have one assertion

Reply:
I have done that sir.

Reply:
Please avoid using `import *` import what you need